source/slang/slang-ir-insts.lua

```diff
@@ -74,13 +80,13 @@ local insts = {
 			},
 			{ Func = { struct_name = "FuncType", hoistable = true } },
```

Comment:
why is this named `Func`. shoudln't it be `FuncType`? how does it differentiate with `IRFunc`?

Reply:
It's named Func because that's the mnemonic used when dumping IR, if it's the same in PascalCase as the c++ struct name then the struct name can be elided.